src/lnp/presentation/tlv.rs

```diff
@@ -50,7 +50,7 @@ pub struct RawRecord(Vec<u8>);
 
 impl EvenOdd for Type {}
 
-#[derive(Debug, Display, Default)]
+#[derive(Debug, Display, Default, Clone)]
```

Comment:
Can you pls move `Clone` to front? I mostly use this pattern to start with `Clone`

Reply:
Removing clone, as not using `Stream` anymore.